message.go

```diff
@@ -707,7 +707,12 @@ func ParseAddressList(fields []interface{}) (addrs []*Address) {
 
 // Format an address list to fields.
 func FormatAddressList(addrs []*Address) (fields []interface{}) {
-	fields = make([]interface{}, len(addrs))
+	addr_len := len(addrs)
```

Comment:
Style: Go uses `camelCase`. But I don't think it's worth it to define a variable here, we can just inline `len(addrs)` at both sites.